src/components/common/Header/Header.jsx

```diff
@@ -19,17 +22,35 @@ const Header = () => {
   const handleContact = () => {
     setIsContactOpen(!isContactOpen);
   };
+
+  const handleGoBack = () => {
+    history.goBack();
+  };
+
   return (
-    <header className="header">
-      <div
-        className="header__icon-container"
-        role="button"
-        onClick={handleClick}
-        onKeyPress={handleClick}
-        tabIndex={0}
-      >
-        <img src={isOpen ? close : hamburger} alt="menu icon" className="header__icon" />
-      </div>
+    <header className={`header ${location.pathname == '/new-target' ? 'bg-lb' : 'bg-w'}`}>
```

Comment:
WDYT if you replace the `/new-target` line with a constant variable? In order to avoid typos or  possible errors in the future.